spec/factories.rb

```diff
@@ -6,4 +6,8 @@
     full_name { 'Testereck Usernder' }
     gender { 2 }
   end
+
+  factory :confirmed_user, parent: :user do
```

Comment:
I would rather go with a trait for this https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#traits

You can mix traits easily, it seems harder to mix factories